recipes/polly/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set version = "15.0.7" %}
+{% set major_ver = version.split(".")[0] %}
+
+package:
+  name: polly
+  version: {{ version }}
+
+source:
+  url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-project-{{ version.replace(".rc", "rc") }}.src.tar.xz
+  sha256: cf3c2a1152ed7a99866bd7f12c24528ada6d9f6336afb7a08416938762004c9f
```

Changing the version requires an updated hash
```suggestion
  sha256: 8b5fcb24b4128cf04df1b0b9410ce8b1a729cb3c544e6da885d234280dedeac6
```

---

Yes, I realized it just now. Thanks!

---

Seems I commented just as your commit arrived. 🙃